first_validation.py

```diff
@@ -48,13 +50,24 @@ def check_hashes(hash_fname):
     # Directory containing hash filenames file.
     data_dir = hash_pth.parent
     # Read in text for hash filename
+    hash_file_contents = hash_pth.read_text()
+
     # Split into lines.
+    lst_of_splitted_lines = hash_file_contents.splitlines()
     # For each line:
+    for line in lst_of_splitted_lines:
         # Split each line into expected_hash and filename
+        [exp_hash, gap, filename] = line.split(sep=' ')
```

Comment:
Yes - but you don't even need the brackets around the list on the left hand side.  And you can use the default separator (of any number of spaces) to make it a bit simpler, as in:

```suggestion
        exp_hash, filename = line.split()
```